proto/skaffold.proto

```diff
@@ -560,6 +563,10 @@ enum SuggestionCode {
     // Check if docker is running
     CHECK_DOCKER_RUNNING = 106;
 
+    // Deploy error suggestion codes
+
+    // Kube config error
+    CHECK_KUBE_CONFIG = 201;
```

Comment:
The more description suggestion code would be  `CHECK_CLUSTER_CONNECTION`

Reply:
Renamed.